source/cloud/cloud-integrations/cloud-slash-commands.rst

```diff
@@ -188,4 +188,4 @@ Known Slack compatibility issues
 Troubleshooting
 ---------------
 
-See `developer documentation <https://developers.mattermost.com/integrate/slash-commands/>`__ for troubleshooting, or `join the Mattermost user community for help <https://mattermost.com/pl/default-ask-mattermost-community/>`_.
+See `developer documentation <https://developers.mattermost.com/integrate/slash-commands>`__ for troubleshooting, or `join the Mattermost user community for help <https://mattermost.com/pl/default-ask-mattermost-community/>`_.
```

Comment:
Should the second URL have a trailing slash?

Reply:
It appears to resolve with and without the trailing slash.